### PR TITLE
Upd rst cb

### PR DESCRIPTION
**Pull request by aschroed**
- Updated some of the development tips documentation to reflect the refactor
- Added argument to rst code directives that were missing them (prevented the code blocks from showing on readthedocs)
- Updated dependency for dcic utils to 1.83 or more to fix rds snapshot check error
- patch version bump

### CODE REVIEW

File: docs/source/deployment.rst [hunk 1]
```diff
@@ -17,14 +17,14 @@ You can also run a local deployment of the dev application by running 'chalice l
 Automatic deployments
 ---------------------
 
-The Github repository for Chalice is outfitted with Travis CI testing and will deploy automatically when code is merged into the production branch. When you merge into production and tests pass, automatic deployment to the prod stage will occur.
+The Github repository for Chalice is outfitted with Github Actions and can be deployed when code is merged into the production branch by running the action available from the `Action <https://github.com/4dn-dcic/foursight/actions>`_ tab on GitHub.
```
Comment (andreacosolo) [inline docs/source/deployment.rst:20]:
```suggestion
The Github repository for Chalice is outfitted with Github Actions and can be deployed when code is merged into the master branch by running the action available from the `Action <https://github.com/4dn-dcic/foursight/actions>`_ tab on GitHub.
```

File: docs/source/development_tips.rst [hunk 2]
```diff
@@ -143,14 +150,14 @@ In some cases you may not want your check to run on a CRON type schedule but sti
...
+.. code-block:: JSON
 
    {
-       "my_first_check": {
-           "title": "My first check",
-           "group": "Awesome test checks",
-           "schedule": {},
-           "display": ["data", "staging"]
-       }
+      "my_first_check": {
```
Comment (andreacosolo) [inline docs/source/development_tips.rst:156]: Indentation looked fine in the previous version.

**Review by andreacosolo — COMMENTED** [review]
Just a couple of minor edit suggestions, but otherwise it looks fine to me.

File: docs/source/development_tips.rst (hunk 2 above)
Comment (aschroed) [reply docs/source/development_tips.rst:156]: fixed

File: docs/source/deployment.rst (hunk 1 above)
Comment (aschroed) [reply docs/source/deployment.rst]: changed.

**Pull request merged**